docs/en/release.md

```diff
@@ -0,0 +1,353 @@
+Is this correct? (y/N) y
+
+# Fill in your personal information. It is recommended to fill in your Apache account information and then enter O to confirm.
+GnuPG needs to construct a user ID to identify your key.
+
+Real name: Zhiguo Wu
+Email address: wuzhiguo@apache.org
+Comment: CODE SIGNING KEY
+You selected this USER-ID:
+    "Zhiguo Wu (CODE SIGNING KEY) <wuzhiguo@apache.org>"
+
+Change (N)ame, (C)omment, (E)mail or (O)kay/(Q)uit? O
+
+We need to generate a lot of random bytes. It is a good idea to perform some other action (type on the keyboard, move the mouse, utilize the disks) during the prime generation; this gives the random number generator a better chance to gain enough entropy.
+
+# The key is successfully created. Please save it properly.
+gpg: directory '/Users/wuzhiguo/.gnupg/openpgp-revocs.d' created
+gpg: revocation certificate stored as '/Users/wuzhiguo/.gnupg/openpgp-revocs.d/D3D5CF25076873C9AA068C0D8F062A5450E25685.rev'
+public and secret key created and signed.
+
+pub   rsa4096 2023-02-21 [SC]
+      D3D5CF25076873C9AA068C0D8F062A5450E25685
+uid           [ultimate] Zhiguo Wu (CODE SIGNING KEY) <wuzhiguo@apache.org>
+sub   rsa4096 2023-02-21 [E]
+```
+
+#### View the Key
+```shell
+$ gpg --list-keys
+
+gpg: checking the trustdb
+gpg: marginals needed: 3  completes needed: 1  trust model: pgp
+gpg: depth: 0  valid:   1  signed:   0  trust: 0-, 0q, 0n, 0m, 0f, 1u
+[keyboxd]
+---------
+pub   rsa4096 2023-02-21 [SC]
+      D3D5CF25076873C9AA068C0D8F062A5450E25685
+uid           [ultimate] Zhiguo Wu (CODE SIGNING KEY) <wuzhiguo@apache.org>
+sub   rsa4096 2023-02-21 [E]
+```
+
+#### Upload the Key
+Visit the [public server](http://keyserver.ubuntu.com/) to upload the key or run the following command to upload it:
+```shell
+$ gpg --keyserver keyserver.ubuntu.com --send-key D3D5CF25076873C9AA068C0D8F062A5450E25685
+```
+
+#### View the Key
+Visit the [public server](http://keyserver.ubuntu.com/) to search for your key or run the following command to view the upload result:
+```shell
+$ gpg --keyserver keyserver.ubuntu.com --search-keys D3D5CF25076873C9AA068C0D8F062A5450E25685
+```
+
+### SVN
+Apache uses SVN to manage release packages.
+
+#### Pull the Directory
+```bash
+$ mkdir -p ~/apache/dev
+$ mkdir -p ~/apache/release
+$ cd ~/apache/dev
+$ svn co https://dist.apache.org/repos/dist/dev/bigtop
+$ cd ~/apache/release
+$ svn co https://dist.apache.org/repos/dist/release/bigtop
+```
+
+#### Upload the Key
+Please note to change the name below to the one you used when generating the key:
+```bash
+$ cd ~/apache/release/bigtop
+$ (gpg --list-sigs Zhiguo Wu && gpg --armor --export Zhiguo Wu) >> KEYS
+$ svn commit -m "Adding Zhiguo Wu's code signing key"
+```
+
+## Start the Release
+### Release Notes
+Since the project is managed by `Apache Jira`, you can easily obtain the corresponding release notes.
+* Go to the [Bigtop Jira](https://issues.apache.org/jira/projects/BIGTOP) page.
+* Click the `Release` menu on the left.
```

Comment:
Thanks for writing this @kevinw66, just one comment.
Before generating release notes, release manager should review if all JIRA issues addressed in this release have the correct fixVersion and unrelated issues don't have that fixVersion, as mentioned [here](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=27849974#Howtorelease-3.GeneratetheReleaseNotes).

Reply:
@sekikn Just updated!